### ISSUE TITLE
[FEATURE] <Change Backgrounds>

### ISSUE THREAD

**Issue by CBID2** [EddieHubCommunity/LinkFree#2125]
### Description

Hi there! I was going through [this review of Linktree](https://www.wanderschool.com/2020/08/14/should-you-use-linktree-the-pros-and-cons/). It mentioned that the **only** pro users are given the option to change their background. I think what can make Linkfree stand out is giving a similar option for free. What do you think?

**Comment by money8203** [comment]
@CBID2 It's a great idea, we should definitely try it out.

**Comment by money8203** [comment]
Yeah sure! ping me whenever you are stuck or needed any kind of guidance.

**Comment by CBID2** [comment]
Great thanks

**Comment by CBID2** [comment]
Hey @money8203! :) Here's a [rough draft of my solution](https://codepen.io/CB_ID2/pen/VwdyKbm). I followed [this tutorial] ( https://www.geeksforgeeks.org/design-background-color-changer-using-html-css-and-javascript/) to create it. For some reason, the CSS and JS is not working.

**Comment by eddiejaoude** [comment]
This requires a lot of thought as it could make someones page unusable if the colours of the background image clashes with the buttons

I think we will close this for now as we have other more requested features
